### PR TITLE
Abolotski miniproject

### PR DESCRIPTION
**Pull request by asherbolotski**
put my limelight miniproject (which is smokey) into my camp folder

### CODE REVIEW

**Comment by mmcadamsms** [comment]
I just built it.  Does it run?

File: camp/abolotski/smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java [hunk 1]
```diff
@@ -0,0 +1,968 @@
...
+  public void zeroRobotPosition()
+  {
+    this.setRobotPosition(new Pose2d(0,0,Rotation2d.fromDegrees(0)));
+  }
+
+  /**
+   * a method that updates the robot position with vision
+   */
+  private void addVisionMeasurement(Pose2d RobotPos, double Timestamp){
+    VisionMeasurement visionMeasurement = cameraSubsystem.getVisionPosition();
```
Comment (mmcadamsms) [inline camp/abolotski/smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java]: This does not have the change I merged to your branch over in ChargedUp2023

Comment (mmcadamsms) [reply camp/abolotski/smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java]: image

Comment (asherbolotski) [reply camp/abolotski/smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java]: fixed

**Review by mmcadamsms — COMMENTED** [review]
There are some binary files in this PR.  Please remove or provide a reason as to why they are necessary. (.jar, .ds-store, etc.)

**Comment by asherbolotski** [comment]
top level .gitignore now includes DS_Store and weeded out the existing DS_Store files from the repo. Remaining .jar file is just the gradle wrapper that is required.

**Review by mmcadamsms — APPROVED** [review]
Thanks for the changes ... commit it.

**Pull request merged**